### ISSUE TITLE
Status Bar Tile

### ISSUE THREAD

**Issue by mehcode** [mehcode/linter-ui-plus#3]
![screen shot 2017-07-10 at 3 41 53 am](https://user-images.githubusercontent.com/753919/28014510-bdb611ce-6521-11e7-8c1f-c2be554b6774.png)

Original tile from `linter-ui-default` is fine except:
 - Should hover / select as a single tile (instead of 3)

 - Use newer icons

 - Do not show `0`. This means the whole tile is hidden if there are no errors, warnings, or infos. This is fine. The only thing it does is toggle the linter panel which is pointless if there are no linter messages. I _can_ see the justification for showing "No Issues" (from linter v1) or showing `0` (from linter v2) as a validation that everything is okay. However, I strongly feel that less is more in this instance. I surveyed several editors and only VS Code shows `0` counts instead of nothing if there are no issues. A large benefit of having the status bar tile hide and show is its a much better visual doorbell rather than just a color change.

 - Test with one-dark-ui from master to make sure it uses newer status bar tile hover ( see https://github.com/atom/one-dark-ui/pull/215 )

**Comment by netizen-ais** [comment]
Somehow liking it, but instead of hiding (maybe as an option) I'm currently _dimming_ 0 values (via stryles.less):
![image](https://user-images.githubusercontent.com/3426685/28015416-2a1eca9c-6570-11e7-8c4c-d0d7f80a89f1.png)

I like it this way because I can then set my line:column as last item in left status bar, without it jumping position because previous elements changing size.

**Comment by mehcode** [comment]
You make a good point with the status bar tiles "jumping". 🤔

**Comment by mehcode** [comment]
Decided to go with a slight fade.

![screen shot 2017-07-12 at 1 14 44 am](https://user-images.githubusercontent.com/753919/28110099-1fdd9766-66a6-11e7-86eb-ca2d73befc34.png)
![screen shot 2017-07-12 at 1 14 37 am](https://user-images.githubusercontent.com/753919/28110098-1fdd373a-66a6-11e7-9db2-764a5761b5c0.png)
![screen shot 2017-07-12 at 1 14 28 am](https://user-images.githubusercontent.com/753919/28110100-1fe7625a-66a6-11e7-98eb-91c6df97e6cc.png)

I made sure it's easy to change in your stylesheet.

If you want to only show a tile with `>= 0`:

```less
.linter-ui-plus.status-bar .tile[data-count="0"] {
  display: none;
}
```

If you want to only hide `info` if its `0`, but leave the other ones showing at `0` (this is what I'm sticking in my styles.less)

```less
.linter-ui-plus.status-bar .tile-info[data-count="0"] {
  display: none;
}
```

---

FYI This is out in v0.3.x

---

I am aware that there is no click affordance shown. This is similar to the newer status tiles introduced by the new Git/GitHub pacakge and will be "fixed" (for the One themes) in atom v1.20 ( via https://github.com/atom/one-dark-ui/pull/215 ).
